Common/src/Stream/Worker/WorkerStreamWrapper.cs

```diff
@@ -47,38 +47,31 @@ public WorkerStreamWrapper(ILoggerFactory loggerFactory)
     loggerFactory_ = loggerFactory;
   }
 
-  /// <inheritdoc />
-  public sealed override async Task Process(IAsyncStreamReader<ProcessRequest> requestStream,
-                                            IServerStreamWriter<ProcessReply>  responseStream,
-                                            ServerCallContext                  context)
+  public sealed override async Task<ProcessReply> Process(IAsyncStreamReader<ProcessRequest> requestStream,
+                                                          ServerCallContext                  context)
   {
-    var taskHandler = await TaskHandler.Create(requestStream,
-                                               responseStream,
-                                               new Configuration
-                                               {
-                                                 DataChunkMaxSize = 50 * 1024,
-                                               },
-                                               loggerFactory_.CreateLogger<TaskHandler>(),
-                                               context.CancellationToken)
-                                       .ConfigureAwait(false);
-
-    using var _ = logger_.BeginNamedScope("Execute task",
-                                          ("taskId", taskHandler.TaskId),
-                                          ("sessionId", taskHandler.SessionId));
-    logger_.LogDebug("Execute Process");
-    var output = await Process(taskHandler)
-                   .ConfigureAwait(false);
-
-    await responseStream.WriteAsync(new ProcessReply
-                                    {
-                                      Output = output,
-                                    })
-                        .ConfigureAwait(false);
-    if (await requestStream.MoveNext(context.CancellationToken)
-                           .ConfigureAwait(false))
+    Output output;
     {
-      throw new InvalidOperationException("The request stream is expected to be finished.");
+      await using var taskHandler = await TaskHandler.Create(requestStream,
+                                                             new Configuration
+                                                             {
+                                                               DataChunkMaxSize = 50 * 1024,
```

Comment:
will this MaxSize will never change? Why not use a parameter?

Reply:
The value can be hardcoded but it needs to be set up in only one place commonly accessible from everywhere. One good approach is to have it in a `Constants` public static class containing public const values.

An even better approach would be for the worker to obtain this value from the polling agent.

Reply:
There is a class that holds this value. I think I will send it to the worker via the compute request. It has always been there.

Reply:
The configuration was already sent but not used. Now it is